src/cdk-experimental/dialog/dialog-container.ts

```diff
@@ -122,6 +124,8 @@ export class CdkDialogContainer extends BasePortalOutlet implements OnDestroy {
     private _elementRef: ElementRef<HTMLElement>,
     private _focusTrapFactory: FocusTrapFactory,
     private _changeDetectorRef: ChangeDetectorRef,
+    private readonly _checker: InteractivityChecker,
```

Comment:
```suggestion
    private readonly _interactivityChecker: InteractivityChecker,
```
I would use `_interactivityChecker` since `_checker` by itself isn't super specific